### ISSUE TITLE
Checkpoints for TRADES

### ISSUE THREAD

**Issue by VSehwag** [LayneH/self-adaptive-training#2]
Hi, 

Thanks for making the code pubic. I am particularly intrigued by the improvement in robust accuracy (in combination with Trades). This seems a very strong result (could be improved further using https://arxiv.org/abs/1905.13736) and I am really interested in testing it further. Would it be possible for you to make the adversarial trained network's checkpoints public?

**Comment by LayneH** [comment]
Hi,

Thanks for your insterest in our work.  We have made the model checkpoint that obtains 58.03% robust accuracy public available at [Google Drive](https://drive.google.com/file/d/1By8kbU-DvL_vAYHdHkDtganrU88n81TT/view?usp=sharing). And we will update the README accordingly.

**Comment by hongyanz** [comment]
Hi @VSehwag, beyond the accuracy improvement, I want to emphasize that another technical advantage of self-adaptive training + TRADES is that the model never overfits for long training. The epoch that @LayneH shared here is the 78th epoch, but other epochs should give similar performance.

**Comment by fra31** [comment]
Hi,

thanks a lot for sharing the model! I ran [AutoAttack+](https://github.com/fra31/auto-attack) on it and got

clean accuracy: 83.48%
robust accuracy 53.29%

using `eps=0.031` (not `8/255` right?).

**Comment by LayneH** [comment]
Hi @fra31 ,

Thank you for your evaluation! You are right that eps=0.031 in our experiments.
 We will include the result in our leaderborad.
